user-has-many-joined-projects

From: Bryan Larsen <bryan@larsen.st>

And in the User model (remember that User already has an association called `projects` so we need a new name):

SHOW_PATCH
---
 app/models/user.rb |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/app/models/user.rb b/app/models/user.rb
index 6bf0d8b..275b816 100644
--- a/app/models/user.rb
+++ b/app/models/user.rb
@@ -15,6 +15,8 @@ class User < ActiveRecord::Base
   has_many :task_assignments, :dependent => :destroy, :inverse_of => :user
   has_many :tasks, :through => :task_assignments
   has_many :projects, :class_name => "Project", :foreign_key => "owner_id", :inverse_of => :owner
+  has_many :project_memberships, :dependent => :destroy, :inverse_of => :user
+  has_many :joined_projects, :through => :project_memberships, :source => :project
 
   # This gives admin rights and an :active state to the first sign-up.
   # Just remove it if you don't want that
